.env

```diff
@@ -0,0 +1,2 @@
+TAG=latest
+BLOCKSTORE_TYPE=local
```

Comment:
I think these need to be prefixed with `LAKEFS_` to get picked up by viper.. Additionally I'd add
`LAKEFS_STATS_ENABLED=false` if this is to be used by local dev environments

Reply:
Done. Adding the LAKEFS_STATS_ENABLED flag to the workflow since the .env is used by default, so it will be disabled for all.